catalog/opensearch/catalog-opensearch-endpoint/src/main/java/org/codice/ddf/opensearch/query/filter/PolygonSpatialFilter.java

```diff
@@ -14,14 +14,15 @@
 package org.codice.ddf.opensearch.query.filter;
 
 import ddf.catalog.impl.filter.SpatialFilter;
+import org.codice.ddf.opensearch.OpenSearchConstants;
 
 public class PolygonSpatialFilter extends SpatialFilter {
   private String[] latLonAry;
 
   public PolygonSpatialFilter(String lonLatStr) {
     super();
 
-    latLonAry = lonLatStr.split(" |,\\p{Space}?");
+    latLonAry = lonLatStr.split(OpenSearchConstants.POLYGON_LON_LAT_DELIMITER);
```

Comment:
❓Is there a reason the lonLatStr string format was made more restrictive?

Reply:
The spec says that the `polygon` query parameter must be in an exact format:
https://github.com/codice/ddf/blob/3d8784a8d77590b5aaed142d80ea8a68ae4caad0/catalog/opensearch/catalog-opensearch-endpoint/src/main/resources/ddf-os.xml#L62-L64